tests/art_unit.cpp

```diff
@@ -39,8 +38,7 @@ void assert_key_eq(art_key_chunk_t* key1, art_key_chunk_t* key2) {
 class Key {
    public:
     Key(uint64_t key) {
-        // Reverse byte order of the low 6 bytes. Not portable to big-endian
-        // systems!
+        // Store the low 6 bytes of the key in big-endian order.
```

Comment:
This is actually totally portable

Reply:
Yes I think I used a different way to do this before and forgot to update the comment.